tasks/eva_2889/remediate_duplicate_remapped_ss/src/RemediateDuplicateRemappedSS.groovy

```diff
@@ -0,0 +1,312 @@
+        }
+        return svesNotGeneratedByRemapping
+    }
+
+    // Transform a SVE based on the different remapping attributes and get the collection
+    static List<String[]> getPossibleRemappedAllelesForSVE(sveFromSourceAssembly, correspondingRemappingAttributes) {
+        return correspondingRemappingAttributes.collect {remappingAttribute ->
+            def (refFromSourceAssembly, altFromSourceAssembly) =
+            [sveFromSourceAssembly.getReferenceAllele(), sveFromSourceAssembly.getAlternateAllele()]
+            // No transformations necessary when there is no novel reference or reference allele change or strand change
+            if (Objects.isNull(remappingAttribute.referenceAlleleChange) && !remappingAttribute.novelReferenceAllele
+                    && remappingAttribute.strand == "+") {
+                return normalizeAlleles(refFromSourceAssembly, altFromSourceAssembly)
+            }
+            def (transformedRef, transformedAlt) = [refFromSourceAssembly, altFromSourceAssembly]
+            if (remappingAttribute.strand == "-") {
+                (transformedRef, transformedAlt) = [SequenceUtil.reverseComplement(transformedRef),
+                                                    SequenceUtil.reverseComplement(transformedAlt)]
+            }
+            if (Objects.nonNull(remappingAttribute.referenceAlleleChange)) {
+                def (referenceAlleleChangeOriginalRef, referenceAlleleChangeNewRef) = remappingAttribute.referenceAlleleChange.split("-")
+                (transformedRef, transformedAlt) = applyReferenceAlleleChange(transformedRef, transformedAlt,
+                        referenceAlleleChangeOriginalRef, referenceAlleleChangeNewRef)
+            }
+            if (transformedRef == transformedAlt) return null
+            return normalizeAlleles(transformedRef, transformedAlt)
+        }.findAll{refAltPair -> Objects.nonNull(refAltPair)}
+    }
+
+    static def applyReferenceAlleleChange(refFromSourceAssembly, altFromSourceAssembly,
+                                          referenceAlleleChangeOriginalRef, referenceAlleleChangeNewRef) {
+        // Try to check if there were "context" nucleotides added before or after the original reference from the source assembly
+        def matcher = (referenceAlleleChangeOriginalRef =~ /(?<allelesToTheLeft>[A-Za-z]*)${refFromSourceAssembly}(?<allelesToTheRight>[A-Za-z]*)/)
+        if (matcher.matches()) {
+            def (allelesToTheLeft, allelesToTheRight) = [matcher.group("allelesToTheLeft"),
+                                                         matcher.group("allelesToTheRight")]
+            refFromSourceAssembly = "${allelesToTheLeft}${refFromSourceAssembly}${allelesToTheRight}"
+            altFromSourceAssembly = "${allelesToTheLeft}${altFromSourceAssembly}${allelesToTheRight}"
+        }
+
+        refFromSourceAssembly = refFromSourceAssembly.replace(referenceAlleleChangeOriginalRef, referenceAlleleChangeNewRef)
```

Comment:
Yes, we return it unchanged. this shouldn't happen a lot since we are explicitly checking [associatedRS too](https://github.com/EBIvariation/eva-tasks/pull/75/files#diff-e25c8e5dcdfe70edae40bb91b54884571eb108a613210eef16d1dd1575a71759R233).